### PR TITLE
Cluster operation endpoints

### LINKED ISSUE #40: Port cluster ops endpoints
As a user of Commissaire
I'd like the v0 cluster ops endpoints ported to the new infra
so I can use them like I did with the MVP

**Acceptance Criteria**

- Verify http://commissaire.readthedocs.io/en/latest/endpoints.html#cluster-operations-deploy is ported
- Verify http://commissaire.readthedocs.io/en/latest/endpoints.html#cluster-operations-upgrade is ported
- Verify http://commissaire.readthedocs.io/en/latest/endpoints.html#cluster-operations-restart is ported

**Requires**: https://trello.com/c/FpBRdKxo/461-3-multi-host-mgr-2-port-clusterexec-to-new-services to be fully functional (being worked on by @mbarnes)

### PR DESCRIPTION
**Pull request by ashcrow**
Resolves #40.

Requires: https://github.com/projectatomic/commissaire/pull/49

### CODE REVIEW

**Comment by ashcrow** [comment]
[(5) [multi-host-mgr-2] Port cluster ops endpoints](https://trello.com/c/2vRefsbt/502-5-multi-host-mgr-2-port-cluster-ops-endpoints)

**Comment by ashcrow** [comment]
@mbarnes I'd say go ahead and merge. Either you or I can then wire up the RPC call to-do's at will. I should have the next two done today as well :smile:.

**Comment by ashcrow** [comment]
@mbarnes I went a head and reused this one since it had not been merged in yet.

File: src/commissaire_http/handlers/clusters/operations.py [hunk 1]
```diff
@@ -108,3 +110,85 @@ def create_cluster_deploy(message, bus):
...
+
+    try:
+        model = model_cls.new(
+            name=cluster_name,
+            started_at=datetime.datetime.utcnow().isoformat())
+        model._validate()
+
+        # TODO: Hook up service call
+        # if isinstance(model_cls, models.ClusterUpgrade):
+        # elif isinstance(models_cls, models.ClusterRestart):
```
Comment (mbarnes) [inline src/commissaire_http/handlers/clusters/operations.py]: Not wild about case-by-case logic here.  Seems like the caller could pass the appropriate arguments as a closure somehow.  But let's worry about that when we wire this up for real.

**Comment by mbarnes** [comment]
@ashcrow Is the CI failure expected?  Is it because the service calls are omitted?

**Comment by ashcrow** [comment]
@mbarnes hrm, no it's not. I'll look into it.

**Comment by ashcrow** [comment]
bot, retest this please

**Comment by ashcrow** [comment]
@mbarnes must be a flake (maybe it didn't run again?). I made no changes and it passed.

**Comment by mbarnes** [comment]
Okay cool.  Thanks for checking.

**Pull request merged**